test/integration/WireMock.spec.ts

```diff
@@ -53,6 +55,40 @@ describe('Integration with WireMock', () => {
                 );
             });
 
+            it('sets up a stub mapping in wiremock server and expects mapping to be called w/ delay', async () => {
+                const requestBody = {
+                    objectKey: {
+                        intKey: 5,
+                        stringKey: 'stringKey',
+                    },
+                };
+                const testEndpoint = '/test-endpoint';
+                const responseBody = { test: 'testValue' };
+                await mock.register(
+                    {
+                        method: 'POST',
+                        endpoint: testEndpoint,
+                        body: requestBody,
+                    },
+                    {
+                        status: 200,
+                        body: responseBody,
+                    },
+                    {
+                        responseDelay: {
+                            type: DelayType.FIXED,
+                            constantDelay: 1000,
```

Comment:
I would say a 1000ms delay is too long for a test, would 100-150ms work?

Reply:
let me do 300

just want to be sure that it is not just an expected delay but the delay function is actually working